themes/ra-theme/assets/css/quote-box.scss

```diff
@@ -1,19 +1,30 @@
 div.quote-box {
   display: grid;
   grid-auto-flow: column;
-  grid-auto-columns: 1fr;
+  grid-auto-columns: initial;
   column-gap: 2rem;
   align-items: stretch;
   overflow: hidden;
   border-radius: 6px;
-
   font-size: 12px;
   line-height: 1.3em;
-
-  padding: 1rem 2rem;
-  width: 400px;
-  height: 200px;
-
+  padding: 2rem 2rem;
+  width: 600px;
+  height: 100%;
```

Comment:
why is height: 100% needed?

Reply:
I didn't wanted a uniform height and so a image bigger than any acceptable px height is added it won't look stretched.. I will remove it or add a fixed height